generated/java/grpc-google-longrunning-v1/build.gradle

```diff
@@ -10,7 +10,7 @@ apply plugin: 'signing'
 
 description = 'GRPC library for grpc-google-longrunning-v1'
 group = "com.google.api.grpc"
-version = "0.1.23"
+version = "0.1.24"
```

Comment:
Do we want to publish another version of this package? Or are we going to stop publishing it and make people use grpc-google-common-protos?

Reply:
I thought I would just publish this one last version and the remove it afterwards.